memcache.py

```diff
@@ -134,6 +134,7 @@ class Client(threading.local):
     _FLAG_INTEGER = 1 << 1
     _FLAG_LONG = 1 << 2
     _FLAG_COMPRESSED = 1 << 3
+    _FLAG_TEXT = 1 << 4
```

Comment:
Note that adding this new flag breaks environments where 1.58 and 1.59 are running mixed (accidentally or otherwise). Might have been nice to point this out in the release notes, or maybe even bump the major version.